src/rust/aggregator/src/data/user_history.rs

```diff
@@ -150,6 +150,8 @@ impl Data for UserHistory {
                 x.side,
                 x.self_match_behavior,
                 x.restriction,
+                x.price,
+                x.time,
```

Comment:
You need to use transaction version and event index instead of time because (as mentioned in Slack) one block has many transactions with the same time and one transaction may have many orders with the same version.